src/utils/useItisSearch.js

```diff
@@ -59,5 +59,5 @@ export default function useItisSearch(searchKey, maxResults = 50) {
     if (searchKey) fetchData();
   }, [searchKey, maxResults]);
 
-  return { data, loading, error };
+  return { data, loading, error, setError, setLoading };
```

Comment:
What is the use case for an external component telling a request that it is or is not loading?

Reply:
See above:
the error would appear, and loading would just continue to stay true.

Reply:
That is a bug in useItisSearch. I think it should be fixed there.